Description: do not hardcode LLVM_DEFAULT_TARGET_TRIPLE,
  because then building gn with that version of clang fails on any
  architecture that's not x86-64.
  This partially reverts https://chromium.googlesource.com/chromium/src/+/2960c2b,
  which was meant to affect Mac only anyway.
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -1007,11 +1007,6 @@ def main():
       cmake_args.append('-DLLVM_DEFAULT_TARGET_TRIPLE=arm64-apple-darwin')
     else:
       cmake_args.append('-DLLVM_DEFAULT_TARGET_TRIPLE=x86_64-apple-darwin')
-  elif sys.platform.startswith('linux'):
-    cmake_args.extend([
-        '-DLLVM_DEFAULT_TARGET_TRIPLE=x86_64-unknown-linux-gnu',
-        '-DLLVM_ENABLE_PER_TARGET_RUNTIME_DIR=ON',
-    ])
   elif sys.platform == 'win32':
     cmake_args.append('-DLLVM_DEFAULT_TARGET_TRIPLE=x86_64-pc-windows-msvc')
 
